src/nvim/regexp_nfa.c

```diff
@@ -6064,10 +6070,16 @@ static int nfa_regmatch(nfa_regprog_T *prog, nfa_state_T *start, regsubs_T *subm
       break;
 
     // Allow interrupting with CTRL-C.
-    fast_breakcheck();
+    line_breakcheck();
     if (got_int) {
       break;
     }
```

Comment:
`// Check for timeout once in a twenty times to avoid overhead.`
Did you omit this comment from the original patch on purpose?